src/components/sideNavigation/SideNavigationMenuSection.tsx

```diff
@@ -1,10 +1,58 @@
 import * as React from 'react';
+import {SlideY} from '../../animations/SlideY';
+import {Svg} from '../svg/Svg';
+import {ISideNavigationHeaderProps, SideNavigationHeader} from './SideNavigationHeader';
 import {SideNavigationLoadingHeader} from './SideNavigationLoadingHeader';
 
-export const SideNavigationMenuSection = (props: any) =>
-    <div className='block navigation-menu-section'>
-        <SideNavigationLoadingHeader />
-        <div className='navigation-menu-section-items'>
-            {props.children}
-        </div>
-    </div>;
+export interface ISideNavigationSectionOwnProps {
+    header?: ISideNavigationHeaderProps;
+    expandable?: boolean;
+}
+
+export interface ISideNavigationSectionStateProps {
+    expanded?: boolean;
+}
+
+export interface ISideNavigationSectionDispatchProps {
+    onClick?: () => void;
+}
+
+export interface ISideNavigationSectionProps extends ISideNavigationSectionOwnProps, ISideNavigationSectionStateProps, ISideNavigationSectionDispatchProps {}
+
+export class SideNavigationMenuSection extends React.Component<ISideNavigationSectionProps> {
+    private handleClick() {
+        if (this.props.onClick) {
+            this.props.onClick();
+        }
+    }
+
+    render() {
+        return (
+            <div className='block navigation-menu-section' onClick={() => this.handleClick()}>
+                {this.buildHeader()}
+                <SlideY in={!(this.props.expandable && !this.props.expanded)} timeout={350}>
+                    <div className='navigation-menu-section-items'>
+                        {this.props.children}
+                    </div>
+                </SlideY>
+            </div>
+        );
+    }
+
+    private buildHeader(): JSX.Element {
+        if (this.props.header) {
+            let arrow = null;
+            if (this.props.expandable) {
+                arrow = this.props.expanded
+                    ? <Svg svgName='arrow-top-rounded' className='collapsible-arrow icon fill-white' />
+                    : <Svg svgName='arrow-bottom-rounded' className='collapsible-arrow icon fill-white' />;

```

Comment:
You could use a constant for `collapsible-arrow icon fill-white` so that we can make sure they have the same class names.